src/ui/public/notify/partials/toaster.html

```diff
@@ -11,10 +11,17 @@
         <div class="btn-group pull-right toast-controls">
           <button
             type="button"
+            ng-if="notif.timerId"
```

Comment:
Kind of leaking an implementation detail here... what do you think about adding a `isTimed()` function to notif, or something similar?